src/components/PasswordInput/PasswordInput.js

```diff
@@ -0,0 +1,47 @@
+import { InputAdornment } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+
+import TextField from "@material-ui/core/TextField";
+import { RemoveRedEye } from "@material-ui/icons";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+
+const styles = theme => ({
+  eye: {
+    cursor: "pointer"
+  }
+});
+
+const PasswordInput = props => {
+  const [passwordIsMasked, setPasswordIsMasked] = useState(true);
+  const { classes } = props;
+
+  const togglePasswordMask = () => {
+    setPasswordIsMasked(!passwordIsMasked);
+  };
+
+  return (
+    <TextField
+      type={passwordIsMasked ? "password" : "text"}
+      {...props}
+      InputProps={{
+        endAdornment: (
+          <InputAdornment position="end">
+            <RemoveRedEye
+              className={classes.eye}
+              onClick={togglePasswordMask}
+            />
```

Comment:
Maybe use these 2 icons and toggle them according with passwordIsMasked value?
<img width="198" alt="Screenshot 2020-04-15 at 17 54 26" src="https://user-images.githubusercontent.com/9432398/79365164-272c3880-7f42-11ea-8c84-d75a42a86cad.png">

Reply:
nice. :) just did the change